README.md

```diff
@@ -1,106 +1,27 @@
-# SwiftSyntax
+# Swift Syntax
 
-SwiftSyntax is a set of Swift libraries for parsing, inspecting, generating, and transforming Swift source code.
+The swift-syntax package is a set of libraries that work on a source-accurate tree representation of Swift source code, called the SwiftSyntax tree. The SwiftSyntax tree forms the backbone of Swift’s macro system – the macro expansion nodes are represented as SwiftSyntax nodes and a macro generates a SwiftSyntax tree to be inserted into the source file.
 
 ## Documentation
 
 You can read SwiftSyntax’s documentation on [swiftpackageindex.com](https://swiftpackageindex.com/apple/swift-syntax/documentation).
 
 A great way to interactively explore the SwiftSyntax tree of a source file is https://swift-ast-explorer.com, developed by [@kishikawakatsumi](https://github.com/kishikawakatsumi).
 
-## Adding SwiftSyntax as a Dependency
+A set of example usages of swift-syntax can be found in [Examples](Examples).
```

Comment:
I would like to see the macro examples cleaned up a little bit before giving them a top-level highlight in the README.md first.

Reply:
I got you

Reply:
Do you have any areas in mind that could be improved in the macro examples not mentioned in #2169?

Reply:
No, I basically meant the first two open points in https://github.com/apple/swift-syntax/issues/2169

- Make implementations of all macros consistent
- Change nodes modification to use variable setters instead of `with` approach

And maybe adding some more comments that explain what’s being done, not sure how much of those would be needed.